.gitignore

```
@@ -8,3 +8,6 @@ target/
 
 # MSVC Windows builds of rustc generate these, which store debugging information
 *.pdb
+
+# Circuits
+circuits/out
```

This would ignore any subdirectory like this. I think you want only the one at the root:

```suggestion
/circuits/out
```